### ISSUE TITLE
Formatting Proof view

### ISSUE THREAD

**Issue by oddcoder** [coq-community/vscoq#749]
Legacy vscoq allows proofs to be formatted nicely, But this is no the case for the current vscoq as shown below:

![image](https://github.com/coq-community/vscoq/assets/10424605/6dcb6582-0775-4636-97a7-d7090cac3639)

This is makes it extremely hard to follow computer science related proofs as the induction principles generated are very large and wordy. I often find myself having to copy paste it to a new tab and reformatting it by hand just to understand what I am trying to do.

**Comment by rtetley** [comment]
Hi and thanks for reporting ! Coud you give me an example to work with ? As far as I remember we had fixed the formatting issues when we switched to using ppstrings.

**Comment by Lee-Janggun** [comment]
https://gitlab.mpi-sws.org/iris/examples should work. Before, both the coq context and the proof goals were formatted nicely with good indentations. However, now there are no indentations.

**Comment by gares** [comment]
Bug:
![Screenshot from 2024-03-22 10-33-51](https://github.com/coq-community/vscoq/assets/1013846/e6f193a3-4564-46f8-aad4-4f771dd9bde8)
Expected:
![Screenshot from 2024-03-22 10-34-23](https://github.com/coq-community/vscoq/assets/1013846/e878fdae-2214-4f4a-8a5d-1f2dd004e925)
Documentation of the box module Coq uses behind the scenes:
https://v2.ocaml.org/api/Format.html
